.editorconfig

```diff
@@ -7,12 +7,16 @@ trim_trailing_whitespace = true
 end_of_line = lf
 insert_final_newline = true
 
-[{.buildkite/hooks/**,*.sh,*.yml,*.yaml}]
+[{*.sh,*.yml,*.yaml}]
```

Comment:
This was changed recently we should probably rebase on master to make sure we pick up the most appropriate changes.